plugin-server/src/main/ingestion-queues/session-recording/utils.ts

```diff
@@ -337,13 +337,13 @@ export const parseKafkaBatch = async (
         const newEventsByWindowId: IncomingRecordingMessage['eventsByWindowId'] = Object.entries(
             existingMessage.eventsByWindowId
         ).reduce((acc, [windowId, events]) => {
-            acc[windowId] = [...events]
+            acc[windowId] = ([] as RRWebEvent[]).concat(events)
```

Comment:
we don't need this bit here as those spread events aren't going on the call stack